### PR TITLE
[codepath] Todo app

### PR DESCRIPTION
**Pull request by wnnwng**
/cc @codepathreview @codepath 

I thought that the prework was done on 5/29 (the email had the wrong date) so haven't had time to get through the extra stuff yet, although I plan to! I'm aiming for the 2nd session anyway, so hopefully this is ok?

### CODE REVIEW

File: SimpleTodo/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+The following **additional** features are implemented:
+
+* [ ]
+
+## Video Walkthrough
+
+Here's a walkthrough of implemented user stories:
+
+<img src='http://imgur.com/MemzI22' title='Video Walkthrough' width='' alt='Video Walkthrough' />
```
Comment (nesquena) [inline SimpleTodo/README.md]: Use <http://i.imgur.com/MemzI22.gif> so the GIF can display inline

**Comment by nesquena** [comment]
Winnie,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The program is project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer to learn all practical Android development and best practices as quickly as possible. We also provide light code feedback for each submitted project once the program begins.

Now that you finished the required parts of this app, the next step is to **work on a few [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo)**. We encourage you to focus your efforts on either improving the user interface or enhancing the functionality of the app. This course is project-based and the optional extensions on each project are often the most valuable learnings since they dive deeper into common Android use cases.

Ping us here again as you you submit extensions. We are accepting people in large part based on the quality of their pre-works until the class is filled up. Let me know if you have any questions in the meantime.

**Comment by nesquena** [comment]
> I thought that the prework was done on 5/29 (the email had the wrong date) so haven't had time to get through the extra stuff yet, although I plan to! I'm aiming for the 2nd session anyway, so hopefully this is ok?

Sorry for the confusion on the date. When you say you are planning on the 2nd session, do you mean you'd prefer not to participate in the sessions starting soon in May?

**Comment by wnnwng** [comment]
@nesquena I'm participating in Airbnb's internal airbnb bootcamp, which will be happening throughout the year. The first one is in May (but i can't make it because of the timing), so I was planning on doing the one after that. I believe these sessions are separate from the ones Codepath advertises on it's website

**Comment by nesquena** [comment]
@wnnwng Okay thanks for confirming and yes you are correct that these sessions you applied for are happening as a part of the internal airbnb bootcamp starting up. I'll make a note that you'd like to participate in the next bootcamp after the pilot in May.

**Comment by wnnwng** [comment]
/cc @codepathreview @codepath

^ Submission for August 1st Airbnb android bootcamp. Originally thought I'd be doing the May one, but had conflicting vacation plans, hence the resubmission

Also added:
SQLite-backed storage
Using DialogFragment

Working on:
Custom adapter

**Comment by nesquena** [comment]
Hi Winnie,

Yes I remember chatting with you last time. Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer to learn all practical Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program begins.

Great to see you were able to complete a few extension tasks as well. The extension tasks available on each project are often the most valuable learnings since they dive deeper into common real-world Android use cases.

Your application will be passed along to the organizers for final admissions. You should hear back again soon. I'd encourage you to continue [working on app extensions](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo). Let me know if you have any questions in the meantime.

**Comment by nesquena** [comment]
Also please feel free to merge this into master :)

**Comment by wnnwng** [comment]
@nesquena Cool! should i tag the same people for future extensions?

**Comment by nesquena** [comment]
No need to tag us again, just reply back to this issue after updating the README and we'll all get notified.

**Pull request merged**